charts/team-ns/templates/argocd/argocd-project.yaml

```diff
@@ -0,0 +1,81 @@
+{{- $v := .Values }}
+{{- $a := $v.apps.argocd }}
+{{- if and (not (eq $v.teamId "admin")) $a.enabled }}
+apiVersion: argoproj.io/v1alpha1
+kind: AppProject
+metadata:
+  name: team-{{ $v.teamId }}
+  namespace: argocd
+  # Finalizer that ensures that project is not deleted until it is not referenced by any application
+  finalizers:
+    - resources-finalizer.argocd.argoproj.io
+spec:
+  description: ArgoCD project for team {{ $v.teamId }}
+  # Allow manifests to deploy from any Git repos
+  sourceRepos:
+  - https://gitea.{{ $v.cluster.domainSuffix }}/otomi/team-{{ $v.teamId }}-argocd.git
+  # Only permit applications to deploy to the guestbook namespace in the same cluster
+  destinations:
+  - namespace: team-{{ $v.teamId }}
+    server: https://kubernetes.default.svc
+  # Deny all cluster-scoped resources from being created
+  clusterResourceWhitelist: []
+  # - group: ''
+  #   kind: Namespace
+  # Allow all namespaced-scoped resources to be created, except for ResourceQuota, LimitRange, NetworkPolicy
+  namespaceResourceBlacklist:
```

Comment:
How about istio resources like VirtualService, Sidecar, Gateway

Reply:
Alright, added

Reply:
Hmmm. Wrong spot. Should be handled by gatekeeper

Reply:
will keep the list for now, but should be moved to gk